git-fs.c

```diff
@@ -27,6 +29,20 @@ bool enable_debug = 0;
 
 #define error(...) fprintf(stderr, __VA_ARGS__)
 
+// Dump a stacktrace to stderr
+static void dump_trace(int signum) {
+	error("Segmentation fault\n");
+	void * buffer[255];
+	const int calls = backtrace(buffer, sizeof(buffer) / sizeof(void *));
```

Comment:
From the man page:

    The size argument specifies the maximum number of addresses that can be stored in buffer.